imap_processing/cdf/utils.py

```diff
@@ -72,6 +79,15 @@ def load_cdf(file_path: Path, **kwargs: dict) -> xr.Dataset:
         if isinstance(value, list) and len(value) == 1:
             dataset.attrs[attribute] = value[0]
 
+    # Remove attributes specific to xarray plotting from vars and coords
+    # TODO: This can be removed if/when feature is added to cdf_to_xarray to
+    #      make adding these attributes optional
+    if remove_xarray_attrs:
+        xarray_attrs = [v for k, v in ISTP_TO_XARRAY_ATTRS.items()]
+        for _, data_array in ChainMap(dataset.data_vars, dataset.coords).items():
+            for key in xarray_attrs:
+                data_array.attrs.pop(key, None)
```

Comment:
I think this might be able to be simplified and not need to use ChainMap.
```suggestion
        for key in (dataset.data_vars | dataset.coords).keys():
            for xarray_key in ISTP_TO_XARRAY_ATTRS.values():
                dataset["key"].attrs.pop(xarray_key, None)
```

Reply:
Thanks for the suggestion. It almost worked but lead me to finding that the `xarray.Dataset.variables` attribute is a lower level access to a dictionary of all coordinates and variables. `Dataset.data_vars` and `Dataset.coords` return different typed objects and so the pipe operator does not work.

Reply:
Awesome, even better! Good to know there is that other attribute, thanks for digging in :)